### ISSUE TITLE
v0.11.1 installs image versions 0.11.0 by default

### ISSUE THREAD

**Issue by mkolesnik** [submariner-io/submariner-operator#1818]
**What happened**:
```
[root@9ca498e2e26e shipyard]# subctl show versions --kubeconfig output/kubeconfigs/kind-config-cluster1
Cluster "cluster1"
COMPONENT                       REPOSITORY                                            VERSION         
submariner                      quay.io/submariner                                    0.11.0          
submariner-operator             quay.io/submariner                                    0.11.0          
service-discovery               quay.io/submariner                                    0.11.0          
 ✓ Showing versions
```

**What you expected to happen**:
```
[root@9ca498e2e26e shipyard]# subctl show versions --kubeconfig output/kubeconfigs/kind-config-cluster1
Cluster "cluster1"
COMPONENT                       REPOSITORY                                            VERSION         
submariner                      quay.io/submariner                                    0.11.1          
submariner-operator             quay.io/submariner                                    0.11.1          
service-discovery               quay.io/submariner                                    0.11.1          
 ✓ Showing versions
```

**How to reproduce it (as minimally and precisely as possible)**:
```
make clusters
curl -sL https://get.submariner.io | VERSION=0.11.1 bash
subctl deploy-broker --kubeconfig output/kubeconfigs/kind-config-cluster1
subctl join --kubeconfig output/kubeconfigs/kind-config-cluster1 broker-info.subm
subctl show versions --kubeconfig output/kubeconfigs/kind-config-cluster1
```

**Anything else we need to know?**:
This will probably necessitate a bump to 0.11.2 as we dont want to overwrite existing 0.11.1 tags, so technically subctl version 0.11.2 should install 0.11.2

Best and easiest way to do so is simply initiate a 0.11.2 release since image version updating logic was fixed by https://github.com/submariner-io/releases/pull/273

**Environment**:
- Diagnose information (use `subctl diagnose all`):
- Gather information (use `subctl gather`)
- Cloud provider or hardware configuration:
- Install tools:
- Others:

**Comment by dfarrell07** [comment]
Do we know what caused this?

**Comment by nyechiel** [comment]
This is now fixed with 0.11.2.
